### PR TITLE
Adding libet-intentional-binding plugin

### PR DESCRIPTION
**Pull request by kinleyid**
The code in the "examples" folder is implemented [here](https://kinleyid.github.io/rsrch/libet-intentional-binding), and a demonstration that this plugin produces meaningful data can be found [here](https://doi.org/10.1016/j.concog.2021.103217).

### CODE REVIEW

**Comment by bjoluc** [comment]
Thanks for the contribution @kinleyid!

I removed `plugin.json` and replaced it with an updated version of the `package.json` from the `plugin-template` package, and I moved `src/index.js` => `index.js` and updated the reference in `examples/index.html` to comply with `plugin-template`.
Without reading the paper, I don't really understand what I am supposed to do in the example. If you have some time and think it's feasible to add a brief instruction text for the task that might help "let's try what this does" guys like me, feel free to do so :grin: Apart from that, this looks good to me!

File: packages/plugin-libet-intentional-binding/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+### Via NPM
+
+```
+npm install @jspsych-contrib/plugin-libet-intentional-binding
+```
+
+```js
+import jsPsychLibetIntentionalBinding from '@jspsych-contrib/plugin-libet-intentional-binding';
+```
+
```
Comment (bjoluc) [inline packages/plugin-libet-intentional-binding/README.md:24]: Oh, and the NPM part of this should be removed as the plugin isn't implemented as a module.

```suggestion
```js
<script src="https://unpkg.com/@jspsych-contrib/plugin-libet-intentional-binding@1.0.0">
```

```

**Comment by kinleyid** [comment]
Thanks for taking a look at this @bjoluc! I've just added instructions to examples/index.html. A copy of the example is [here](https://kinleyid.github.io/rsrch/libet-intentional-binding).

**Review by bjoluc — APPROVED** [review]
Thanks @kinleyid, this looks great now!

**Comment by bjoluc** [comment]
This is [online](https://www.npmjs.com/package/@jspsych-contrib/plugin-libet-intentional-binding) now: https://unpkg.com/@jspsych-contrib/plugin-libet-intentional-binding@1.0.0 :tada:

**Pull request merged**